docs/cli.rst

```diff
@@ -1237,10 +1237,26 @@ facilitates rapid iterations, inspections of any pipeline as well as debugging.
     $ nextflow run nextflow-io/hello -with-tower
 
 - Invoke the nextflow pipeline execution with a custom parameters ``YAML/JSON`` file.
-  The parameters which are specified through this mechanism are merged with the resolved configuration (base configuration and profiles) and only the common fields are overwritten by the ``YAML/JSON`` file.::
 
-    $ nextflow run main.nf -params-file pipeline_params.yml
+  The parameters which are specified through this mechanism are merged with the resolved configuration (base configuration and profiles) and only the common fields are overwritten by the ``YAML/JSON`` file. ::
```

Comment:
```suggestion
  The parameters which are specified through this mechanism are merged with the resolved configuration (base configuration and profiles). The values provided via params file overwrites the ones with the same name in the Nextflow configuration::
```

Reply:
Actually I'd put the sentence "The values provided via params file overwrites the ones with the same name in the Nextflow configuration file" in a tip *after* the example